docs/designers-developers/designers/animation.md

```diff
@@ -0,0 +1,65 @@
+# Animation
+
+Animation can help reinforce a sense of hierarchy and spatial orientation. This document goes into principles you should follow when you add animation.
+
+## Principles
+
+### Point of Origin
+
+- Animation can help anchor an interface element. For example a menu can scale up from the button that opened it.
+- Animation can help give a sense of place; for example a sidebar can animate in from the side, implying it was always hidden off-screen.
+- Design your animations as if you're working with real-world materials. Imagine your user interface elements are made of real materials — when not on screen, where are they? Use animation to help express that.
+
+### Speed
+
+- Animations should never block a user interaction. They should be fast, almost always complete in less than 0.2 seconds.
+- A user should not have to wait for an animation to finish before they can interact.
+- Animations should be performant. Use `transform` CSS properties when you can, these render elements on the GPU, making them smooth. 
+- If an animation can't be made fast & performant, leave it out.
+
+### Simple
+
+- Don't bounce if the material isn't made of rubber.
+- Don't rotate, fold, or animate on a curve. Keep it simple.
```

Comment:
Curious about the reference to animating on a curve, assuming this is referring to an easing curve (not literally a shape animation). I think that can often help an animation feel smoother… any particular "feel" this is intended to discourage?

Reply:
No no, this animation is a cubic bezier for the right easing. I was referring to the curve of the animation path. But the fact that this came up makes me think the language should not clarified 👍

Reply:
@jasmussen Got it that makes a lot more sense :)